dali/dfu/dfurun.cpp

```diff
@@ -1526,7 +1559,7 @@ class CDFUengine: public CInterface, implements IDFUengine
                         if (needrep)
                             feedback.repmode=cProgressReporter::REPbefore;
                         if (foreigncopy)
-                            checkPhysicalFilePermissions(srcFdesc,userdesc,false);
+                            checkForeignFilePermissions(source,userdesc,false);
```

Comment:
looks fine, but unfortunately, we need to keep backward compatibility, by also falling back to perform the legacy check on a the physical file format (i.e. continue to call checkPhysicalFilePermissions if denied on the new semantics.
But it should be a config. option so we can disable these legacy checks (default to off/no checking of legacy in cloud).